src/feed/SubscriptionManager.cpp

```diff
@@ -50,7 +50,7 @@ void
 SubscriptionManager::pubBookChanges(
     ripple::LedgerHeader const& lgrInfo,
     std::vector<data::TransactionAndMetadata> const& transactions
-) const
```

Comment:
Why?

Reply:
This is due to removal of const all over the place in Strand. Kind of a chain reaction.
The change in Strand is consistent with the same changes in Contexts themselves though - we just did not notice this before.